docs/release_notes.md

```diff
@@ -33,6 +36,20 @@
 - Fixed import of the constraint weight arrays on sparse arrays. *AdonisFX-1155*
 - Fixed a bug that caused geometry sliding constraints to not find valid triangles to slide on initialisation. *AdonisFX-1165*
 - Fixed paint flood operation not normalising the weights properly. *AdonisFX-1146*
+- Fixed a bug that occurred when data from the target mesh was missing. *AdonisFX-1183*
+- Fixed a bug that allowed to remove target geometries while painting Slide On Geometry constraints. *AdonisFX-1192*
+- Fixed a bug that allowed to open the AdonisFX Paint Tool for the final mesh of an AdnSkinMerge deformer. *AdonisFX-1198*
+- Fixed AdonisFX Paint Tool initialization of simulated nodes with intermediate nodes. *AdonisFX-1184*
+
+### Improvements
+
+- Allow source nodes of any type in the *Sensors Connection Editor*. *AdonisFX-1189*
```

Comment:
Move this one to the list of "Maya". We probably can remove "Improvements" if there are no other items

Reply:
Moved. There is still one entry in improvements: "- Prevent the use of simulated muscles as geometry targets. *AdonisFX-1200*". But we might want to exposit in other way.